FR #966: Improving dataset metadata (#1464)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/metadata/dataset/focus_dataset.md

> # FOCUS Dataset Id

Reviewer: ```suggestion
# FOCUS Dataset ID
```

The identifier of the FOCUS dataset for which the schema and its data conform to. This indicates which FOCUS dataset the data artifact aligns with, such as "FOCUS Cost and Usage" or "FOCUS Contract."

> FOCUS Dataset ID MUST be provided in the metadata. FOCUS Dataset ID MUST be of type String and MUST NOT contain null values. FOCUS Dataset ID MUST match the Dataset ID of one of the [*FOCUS datasets*](#glossary:FOCUS-dataset) defined in the FOCUS specification.

Reviewer: ```suggestion
FOCUS Dataset ID MUST be provided in the dataset instance metadata. FOCUS Dataset ID MUST be of type String and MUST NOT contain null values. FOCUS Dataset ID MUST match the Dataset ID of one of the [*FOCUS datasets*](#glossary:FOCUS-dataset) defined in the FOCUS specification.
```

# FOCUS Dataset Id

> The name of the FOCUS dataset for which the schema and its data conform to. This indicates which FOCUS dataset the data artifact aligns with, such as "FOCUS Cost and Usage" or "FOCUS Contract."

Reviewer: ```suggestion
The identifier of the FOCUS dataset for which the schema and its data conform to. This indicates which FOCUS dataset the data artifact aligns with, such as "FOCUS Cost and Usage" or "FOCUS Contract."
```

> The name of the FOCUS dataset for which the schema and its data conform to. This indicates which FOCUS dataset the data artifact aligns with, such as "FOCUS Cost and Usage" or "FOCUS Contract."

Reviewer 2: ```suggestion
The name of the FOCUS dataset for which the schema and its data conform. This indicates which FOCUS dataset the data artifact aligns with, such as "FOCUS Cost and Usage" or "FOCUS Contract Commitment".
```

Author: Resolved by irenas suggestion

> FOCUS Dataset MUST be provided in the metadata. FOCUS Dataset MUST be of type String and MUST NOT contain null values. FOCUS Dataset MUST match one of the published [*FOCUS datasets*](#glossary:FOCUS-dataset) of the FOCUS specification.

Reviewer: I suggest we explicitly specify that Dataset IDs must be used. 

```suggestion
FOCUS Dataset MUST be provided in the metadata. FOCUS Dataset MUST be of type String and MUST NOT contain null values. FOCUS Dataset MUST match the DatasetId of one of the [*FOCUS datasets*](#glossary:FOCUS-dataset) defined in the FOCUS specification.
```

> FOCUS Dataset MUST be provided in the metadata. FOCUS Dataset MUST be of type String and MUST NOT contain null values. FOCUS Dataset MUST match one of the published [*FOCUS datasets*](#glossary:FOCUS-dataset) of the FOCUS specification.

Reviewer: Should we also mention FocusVersions? I don’t have a clear picture of the element hierarchy in the Metadata right now, but I’m wondering whether we should, for example, explicitly mention the FocusVersion as part of the condition here.

Author: FOCUS version is part of the schema definition and is already present in schema metadata.

> FOCUS Dataset MUST be provided in the metadata. FOCUS Dataset MUST be of type String and MUST NOT contain null values. FOCUS Dataset MUST match the DatasetId of one of the [*FOCUS datasets*](#glossary:FOCUS-dataset) defined in the FOCUS specification.

Reviewer: ```suggestion
FOCUS Dataset ID MUST be provided in the metadata. FOCUS Dataset ID MUST be of type String and MUST NOT contain null values. FOCUS Dataset ID MUST match the Dataset ID of one of the [*FOCUS datasets*](#glossary:FOCUS-dataset) defined in the FOCUS specification.
```

> FOCUS Dataset MUST be provided in the metadata. FOCUS Dataset MUST be of type String and MUST NOT contain null values. FOCUS Dataset MUST match the DatasetId of one of the [*FOCUS datasets*](#glossary:FOCUS-dataset) defined in the FOCUS specification.

Reviewer 2: ```suggestion
FOCUS Dataset Id MUST be provided in the metadata. FOCUS Dataset Id MUST be of type String and MUST NOT contain null values. FOCUS Dataset Id MUST match the DatasetId of one of the [*FOCUS datasets*](#glossary:FOCUS-dataset) defined in the FOCUS specification.
```

Author: Resolved by irena's suggestion.

## Metadata ID

> FOCUSDataset

Reviewer: Although I previously suggested replacing FOCUSDataset with DatasetId, now I'm having second thoughts 😄 

```suggestion
FocusDataset
```

Author: Check out  the hybrid name now.

> ## Metadata Id

Reviewer: ```suggestion
## Metadata ID
```

> FOCUSDataset

Reviewer: ```suggestion
FocusDatasetId
```

## Metadata Name

> FOCUS Dataset Id

Reviewer: ```suggestion
FOCUS Dataset ID
```
